docs/basics.md

```diff
@@ -29,11 +29,8 @@ $ devenv shell
 Building shell ...
 Entering shell ...
 
+(devenv) $ echo $GREET
 hello
-jq-1.6
```

Comment:
It'd still have to be:
```
$ devenv shell
Building shell ...
Entering shell ...

hello 
jq-1.6

(devenv) $ echo $GREET
hello
```

I really have no idea what @domenkozar was trying to tell us about the unicorns 😂

Reply:
It makes for me no sense to show the packages or what it should be 🤔

Reply:
My point was that the output should match the previous example.

Example: 
```nix
{ pkgs, ... }: # (1)!

{ # (2)!
  env.GREET = "hello"; # (3)!

  packages = [ pkgs.jq ];

  enterShell = ''
    echo $GREET
    jq --version
  ''; # (4)!
}
```
Output:
```
$ devenv shell
Building shell ...
Entering shell ...

hello 
jq-1.6

(devenv) $ echo $GREET
hello
```

> It makes for me no sense to show the packages or what it should be 🤔
I think the idea is to show how to add packages and that they then become available in the shell.

Reply:
is updated :)